### ISSUE TITLE
preprocessing

### ISSUE THREAD

**Issue by 1475963** [1475963/sentence-boundary-detection#4]
* Generate a dataset with less instances because we have lots of them and we need less data during development to iterate faster
* Preprocess sentences, clean them, replace end of sentence punctuation characters by <PUNCT> so the algorithm will consider `?!.` the same way
* Reformat dataset instances, they shouldn't be sentences but character ngrams or word ngram, each ngram will be labeled as a sentence boundary or not
* Generate a training set, a validation set and a test set
